src/hal/user_comps/pi500_vfd/Submakefile

```diff
@@ -0,0 +1,31 @@
+ifdef HAVE_LIBMODBUS3
+
+PI500_SRCS = hal/user_comps/pi500_vfd/pi500_vfd.c
+PI500_CFLAGS = -DDEBUG $(LIBMODBUS_CFLAGS)
+PI500_LIBS = $(LIBMODBUS_LIBS)
+
+$(call TOOBJSDEPS, $(PI500_SRCS)) : EXTRAFLAGS += $(PI500_CFLAGS)
+
+USERSRCS += $(PI500_SRCS)
+
+../bin/pi500_vfd: $(call TOOBJS, $(PI500_SRCS)) ../lib/liblinuxcnchal.so.0 ../lib/liblinuxcncini.so.0
+	$(ECHO) Linking $(notdir $@)
+	$(Q)$(CC) $(LDFLAGS) -o $@ $^ $(PI500_LIBS) 
+
+hal/user_comps/pi500_vfd/pi500_vfd.c: hal/user_comps/pi500_vfd/pi500_vfd.comp ../bin/halcompile
+	$(ECHO) "Preprocessing wj200_vfd.comp"
```

Comment:
I think that this probably shouldn't say WJ200 any more? 

Does this comp actually need its own submakefile? I created a userspace component in .comp format a few months  ago and just had to add it to the /user-comps/Submakefile rather than make a new one.

Reply:
Thanks for the review. You are totally right. The WJ200 shouldn't be there anymore.

Regarding the Submakefile, I think it is more readable than put everything in the same Submakefile. Putting it all into one file is more error prone to changes.

Reply:
I corrected the wj200 into pi500